pkg/app/piped/platformprovider/kubernetes/manifest_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2022 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubernetes
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func TestParseManifests(t *testing.T) {
+	maker := func(name, kind string, metadata map[string]interface{}) Manifest {
+		return Manifest{
+			Key: ResourceKey{
+				APIVersion: "v1",
+				Kind:       kind,
+				Name:       name,
+				Namespace:  "default",
+			},
+			u: &unstructured.Unstructured{
+				Object: map[string]interface{}{
+					"apiVersion": "v1",
+					"kind":       kind,
+					"metadata":   metadata,
+				},
+			},
+		}
+	}
+
+	testcases := []struct {
+		name      string
+		manifests string
+		want      []Manifest
+	}{
+		{
+			name: "empty1",
+		},
+		{
+			name:      "empty2",
+			manifests: "",
+		},
```

Comment:
Nice improvements.
These seem to be the same cases, so I think one or the other is sufficient. wdyt?

Reply:
corrected my mistake 🙏